### PR TITLE
fix: integration test

### PR DESCRIPTION
**Pull request by zixiang2018**
## What this PR does
1. fix `npm run integration:local`
2. fix `npm run e2e:local`
3. use `.env` variables in email sls
4. update `.env.example`'s bucket name to match storage's serverless default `BUCKET_NAME`
5. standardize .js files with handler to use ESM exporting - `export const handler = middy()...` 
6. standardize all .js files handler to remove callback; use async/await (recommended by https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html)
7. turn off `import/prefer-default-export` in eslintrc for (5) to work
8. turn off `prefer-regex-literals` for `test/utils/matcher.js`

## Pipeline
- Refer to this https://console.seed.run/openattestation/opencerts-functions/activity/stages/pr148/

### CODE REVIEW

File: src/email/mailer/mailer.js [hunk 1]
```diff
@@ -15,47 +15,45 @@ const sendRawMail = (transporter, data) =>
...
+        },
+        {
+          filename: FILE_NAME,
+          content
+        }
+      ]
+    });
+  };
 
 module.exports = sendCertificate;
```
Comment (HJunyuan) [inline src/email/mailer/mailer.js:59]: Do we also have to conform to ESM exporting here?

```suggestion
export sendCertificate;
```

File: test/integration/storage.integration.test.js [hunk 2]
```diff
@@ -27,6 +30,7 @@ describe("uploadDocument", () => {
     isValid.mockClear();
   });
   it("should work without queue number", async () => {
+    // expect(verify()).toBe(true);
```
Comment (HJunyuan) [inline test/integration/storage.integration.test.js:33]: Do we need to expect `verify()` here?

File: src/email/mailer/mailer.js (hunk 1 above)
Comment (zixiang2018) [reply src/email/mailer/mailer.js:59]: Let me check as these files (not directly related to the index.js or is a handler) did not throw any errors

Comment (zixiang2018) [reply src/email/mailer/mailer.js:59]: Just checked, we dont have to conform to ESM exporting here. Mainly the serverless handlers are affected by this change.

**Review by HJunyuan — APPROVED** [review]

**Pull request merged**